src/BlockEntities/EnchantingTableEntity.cpp

```diff
@@ -0,0 +1,50 @@
+
+// BlockEntityEnchantingTable.cpp
+
+#include "Globals.h"  // NOTE: MSVC stupidness requires this to be the same across all modules
+
+#include "EnchantingTableEntity.h"
+
+#include "../ClientHandle.h"
+
+
+cEnchantingTableEntity::cEnchantingTableEntity(BLOCKTYPE a_BlockType, NIBBLETYPE a_BlockMeta, Vector3i a_Pos, cWorld * a_World):
+	Super(a_BlockType, a_BlockMeta, a_Pos, a_World),
+	m_CustomName()
```

Comment:
Could you put this member in the constructor too?

Reply:
you meen the custom name? 
Yes i could do that